src/main.ts

```diff
@@ -1,8 +1,21 @@
 import Vue from "vue";
 import router from "./routes";
+import VueMaterial from "vue-material";
 
-import "./styles/main.css";
-import "material-design-lite";
+import "material-design-icons/iconfont/material-icons.css";
+import "vue-material/dist/vue-material.css";
+
+
+Vue.use(VueMaterial);
+Vue.filter('truncate', function(value) {
+    var length = 70;
+
```

Comment:
It allows truncation of text which exceeds a lengh of 60. You can see it in action for the cards on the mainPage.vue file.